module-3/back-end/index.js

```diff
@@ -43,6 +42,16 @@ app.get('/selectAllDocuments', async (req, res) => {
   }
 })
 
+app.get('/selectAllDocuments/:projectId', async (req, res) => {
+  try {
+    const result = await Document.selectAllDocumentsByProject(req.params.projectId)
```

Comment:
I think `selectAllDocumentsByProjectId` would be a better name for this method

Reply:
Make sense